src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -167,13 +167,14 @@ public void shoot () {
     }
 
     public void testSpin () {
-        pidController.setReference(-4000, ControlType.kVelocity);
+        setShooterRpm(4000.0);
         SmartDashboard.putString("Shooting", "Shooting");
     }
 
-    public void setShooterRPM (double rpm) {
+    public void setShooterRpm (double rpm) {
```

Comment:
Please add comments before the Line 174 and/or before Line 176 explaining why we are negating the input rpm value.